spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -549,3 +551,54 @@ foo-log-v2 started           1/1         1G       1G     foo-log-v2.cfapps.io
 
 NOTE: A comprehensive canary analysis along with rolling upgrades will be supported via http://www.spinnaker.io/[Spinnaker]
 in future releases.
+
+[[getting-started-maximum-disk-quota-configuration]]
+== Maximum Disk Quota Configuration
+By default, every application in Cloud Foundry starts with 1G disk quota and this can be adjusted to a default maximum of
+2G. The default maximum can also be overridden up to 10G via Pivotal Cloud Foundry's (PCF) Ops Manager GUI.
+
+This configuration is relevant for Spring Cloud Data Flow because every stream and task deployment is composed of applications
+(typically Spring Boot uber-jar's) and those applications are resolved from a remote maven repository. After resolution,
+the application artifacts are downloaded to the local Maven Repository for caching/reuse. With this happening in the background,
+there is a possibility the default disk quota (_1G_) fills up rapidly; especially, when we are experimenting with streams that
+are made up of unique applications. It is unusual to encounter this in a production setting, but it is possible if you've
+several unique applications in the streaming or batch pipelines. In order to overcome this disk limitation and depending
+on your scaling requirements, you may want to change the default maximum from 2G to 10G or even more. Let's review the steps to
+change the default maximum disk quota allocation.
+
+PCF's Operations Manager Configuration:
+
+From PCF's Ops Manager, Select "**Pivotal Elastic Runtim**e" tile and navigate to "**Application Developer Controls**" tab.
```

Comment:
The "e" in Runtime is not part of the **bold** inline text. Again, a single asterisk is fine in asciidoc